gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 # libraries
 junit = "4.13.2"
-kubernetes-client = "7.1.0"
+kubernetes-client = "7.3.0"
 jackson-core = "2.18.3"
```

Comment:
let's also bump jackson core to 2.19, as kubernets client uses it : https://central.sonatype.com/artifact/io.fabric8/kubernetes-client-api/7.3.0/dependencies